### PR TITLE
Add/google analytics

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
- Google Tag Manager installed

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -48,7 +55,10 @@
 		window.MODE = MODE;
 	</script>
 	<body>
-		<noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (sepandhaghighi) [inline public/index.html:51]: Why did you remove this line?

File: public/index.html [hunk 2]
```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<!-- Google Tag Manager -->
```
Comment (sepandhaghighi) [inline public/index.html:4]: Is it okay to add this script in the `head` section? Most websites suggest adding scripts like this to the end of the file.
Please double-check it

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (AHReccese) [reply public/index.html:4]: image

Comment (AHReccese) [reply public/index.html:4]: I understand, I just followed the instructions.

File: public/index.html (hunk 1 above)
Comment (AHReccese) [reply public/index.html:51]: I thought it might not be very useful, but since it is kind of a rule, I put it back.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**